playwright/pages/settings.ts

```diff
@@ -25,8 +25,9 @@ export class Settings {
      * Open the top left user menu, returning a Locator to the resulting context menu.
      */
     public async openUserMenu(): Promise<Locator> {
-        await this.page.getByRole("button", { name: "User menu" }).click();
         const locator = this.page.locator(".mx_ContextualMenu");
+        if (await locator.isVisible()) return locator;
```

Comment:
does this not risk finding some other context menu which happens to be open?

Reply:
bleh I guess this is why you updated it in a later commit. More fool me for attempting to review commit-by-commit